### PR TITLE
chore: fixed some error reported by eslint

### PR DESCRIPTION
**Pull request by Frodigo**
## Description
I just ran `eslint --fix` command....

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/theme/types/index.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+interface UseContextReturn {
+  $recaptcha: any
```
Comment (bartoszherba) [inline packages/theme/types/index.ts:2]: I am not sure what is this? Using any is a generally bad practice as it is meaningless unless there is a very good reason to use any. Also, I do not see where do you use this interface?

Comment (bartoszherba) [inline packages/theme/types/index.ts:2]: I am not sure what is this? Using any is a generally bad practice as it is meaningless unless there is a very good reason to use any. Also, I do not see where do you use this interface?

Comment (bartoszherba) [inline packages/theme/types/index.ts:2]: I am not sure what is this? Using any is a generally bad practice as it is meaningless unless there is a very good reason to use any. Also, I do not see where do you use this interface?

**Review by bartoszherba — DISMISSED** [review]
@Frodigo One small comment, probably redundant file was included.

File: packages/theme/types/index.ts (hunk 1 above)
Comment (Frodigo) [reply packages/theme/types/index.ts]: @bartoszherba this is a mistake, removed.

**Review by bartoszherba — APPROVED** [review]

**Pull request merged**